packages/files/_test.pony

```diff
@@ -752,7 +752,7 @@ class iso _TestFileMixedWriteQueue is UnitTest
             line3 + "\n"
             line5
             line1
-            line3 + "\n"
+            line3
```

Comment:
Is this change related to the others?

Reply:
Yes, this is due to the `Writer` now coalescing `write()` buffers into one big buffer if the `buffer`s being added are smaller than a cacheline (https://github.com/ponylang/ponyc/pull/2890/files#diff-761128221a9d08227d2ae82f79a30ea2R258).

The fact that the output of `Writer.done()` can be split across multiple `Array[ByteSeq]`s is an implementation detail (one which changes slightly due to the new coalescing behavior of `Writer.write()`).

The test being fixed relies on the `printv()` call adding in `\n` after every individual buffer returned from `Writer.done()` and the new coalescing only produces 1 buffer instead of 2 resulting in the test failing without this fix.